diff --git a/database-engine/src/test/java/edu/csu/chainpage/engine/api/DatabaseCliTest.java b/database-engine/src/test/java/edu/csu/chainpage/engine/api/DatabaseCliTest.java
index 6c8bb64..90f344d 100644
--- a/database-engine/src/test/java/edu/csu/chainpage/engine/api/DatabaseCliTest.java
+++ b/database-engine/src/test/java/edu/csu/chainpage/engine/api/DatabaseCliTest.java
@@ -39,7 +39,7 @@ class DatabaseCliTest {
                 )))
         );
 
-        assertEquals("Query OK, 1 row affected\n", output.toString(StandardCharsets.UTF_8));
+        assertEquals("Query OK, 1 row affected" + System.lineSeparator(), output.toString(StandardCharsets.UTF_8));
     }
 
     @Test
